### PR TITLE
Update module from template

### PR DESCRIPTION
**Pull request by zied-elouaer**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Internal: Added test suite
```
Comment (waxb) [inline CHANGELOG.md:12]: are we using `Internal:`?

File: test/go.mod [hunk 2]
```diff
@@ -0,0 +1,8 @@
+module github.com/mineiros-io/terraform-module-template/test
```
Comment (waxb) [inline test/go.mod:1]: please run `make template/adjust`

**Review by waxb — CHANGES_REQUESTED** [review]
please move 
```
section {
        title = "Module Configuration"
```
below
```
      section {
        title = "Main Resource Configuration"
```

File: CHANGELOG.md (hunk 1 above)
Comment (zied-elouaer) [reply CHANGELOG.md:12]: I got that from https://github.com/mineiros-io/terraform-google-project-iam/blob/main/CHANGELOG.md

File: test/go.mod (hunk 2 above)
Comment (zied-elouaer) [reply test/go.mod]: done

**Comment by zied-elouaer** [comment]
> please move
> 
> ```
> section {
>         title = "Module Configuration"
> ```
> 
> below
> 
> ```
>       section {
>         title = "Main Resource Configuration"
> ```

Done.

File: CHANGELOG.md (hunk 1 above)
Comment (waxb) [reply CHANGELOG.md:12]: nice, will use it from now on!

**Review by waxb — DISMISSED** [review]
lgtm

File: main.tf [hunk 3]
```diff
@@ -11,6 +10,7 @@ resource "google_project_service" "service" {
 
   timeouts {
     create = try(var.timeouts.create, "15m")
+    read   = try(var.timeouts.read, "10m")
     update = try(var.timeouts.update, "10m")
     delete = try(var.timeouts.delete, "10m")
```
Comment (mariux) [inline main.tf:15]:
```suggestion
    create = try(var.timeouts.create, null)
    read   = try(var.timeouts.read, null)
    update = try(var.timeouts.update, null)
    delete = try(var.timeouts.delete, null)
```

File: CHANGELOG.md (hunk 1 above)
Comment (mariux) [inline CHANGELOG.md:12]:
```suggestion
- Add unit tests
```

**Review by mariux — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (waxb) [reply CHANGELOG.md]: Screenshot 2022-10-10 at 11 08 30
@mariux i guess not then?

**Review by mariux — DISMISSED** [review]
lgtm

File: test/terramate_google.tm.hcl [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+      project_id = var.gcp_project
+    }
+
+    data "google_organization" "org" {
+      count = var.gcp_org_domain != null ? 1 : 0
+
+      domain = var.gcp_org_domain
+    }
+
+    data "google_billing_account" "billing_account" {
```
Comment (waxb) [inline test/terramate_google.tm.hcl:74]: this has been reverted and added as a variable

**Review by waxb — APPROVED** [review]
lgtm

**Pull request merged**